light-client/src/store.rs

```diff
@@ -37,21 +38,35 @@ pub trait LightStore: std::fmt::Debug + Send {
     /// Get an iterator of all light blocks with the given status.
     fn all(&self, status: Status) -> Box<dyn Iterator<Item = LightBlock>>;
 
+    /// Get a block at a given hieht whatever its verification status
+    fn get_any(&self, height: Height) -> Option<(LightBlock, Status)> {
```

Comment:
it's not really "any" because that would contain `Failed` too?

Reply:
No, will remove it!

Reply:
Oops, wrong thread.

Good point, will find a better name.